src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -235,7 +233,9 @@ const MultiscaleImageLayer = class extends CompositeLayer {
           !transparentColor,
         pickable: { type: 'boolean', value: true, compare: true },
         onHover,
-        onClick
+        onClick,
+        // Background image is nicest when LINEAR in my opinion.
+        interpolation: interpolation || INTERPOLATION_MODES.LINEAR
```

Comment:
Hmm, why not leave `defaultProps.interpolation` as `INTERPOLATIONS_MODES.LINEAR` like it was before?